src-tauri/tauri.conf.json

```
@@ -17,6 +17,9 @@
         "all": false,
         "open": true
       },
+      "dialog": {
+        "all": true
+      },
```

```suggestion
      "dialog": {
        "open": true
      },
```

I think you only ever call the `dialog.open()` function right?